glide.yaml

```diff
@@ -2,3 +2,7 @@ package: github.com/giantswarm/microstorage
 import:
 - package: github.com/giantswarm/microerror
   version: master
+# This is only used in storagetest package so it will not appear in your
+# project dependency, unless you use storagetest.
```

Comment:
This only holds true if it would be put into the test import block, doesn't it? This here is placed into the usual import block.